taxonomies/categories.txt

```diff
@@ -32726,7 +32747,7 @@ ciqual_food_name:en:Yogurt, goat's milk, plain, around 5% fat
 en:Stirred yogurts
 de:cremiger Joghurt
 fi:sekoitetut jugurtit
-fr:Yaourts brassés, Yaourt brassé
+fr:Yaourts brassés, Yaourt brassé, brasse
```

Comment:
"brasse" ? Unaccented values are added automatically, there is no need to add them.

Reply:
Actually, I found it here: https://fr.openfoodfacts.org/ciqual-food-name/unknown/categorie/yaourts/categories

Reply:
![image](https://user-images.githubusercontent.com/53179327/81910902-d6edd800-95cc-11ea-8a15-1c98bbb88837.png)